CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.7)
+project(lbrycrd_clion) # not expecting a full compile -- just something that allows clion syntax checking
+
```

Comment:
maybe make this more explicit, so people don't get confused what this file is for ?  
"Do not use for full compile. This is for CLion syntax checking only".